src/transfer.rs

```diff
@@ -454,13 +500,15 @@ where
             .get(header::ETAG)
             .and_then(|v| v.to_str().ok());
 
-        let id = self.0.next_id();
+        let id = self.inner.next_id();
 
         // Check to see if we can download the file with blocks
+        // For block downloads, the server must have responded with the resource size,
+        // accept byte ranges, and provided a strong etag.
         let result = match (file_size, accept_ranges, etag) {
-            (Some(file_size), Some("bytes"), Some(etag)) => {
+            (Some(file_size), Some("bytes"), Some(etag)) if !etag.starts_with("W/") => {
```

Comment:
What does the ETAG starting with `W/` mean? I'm curious if we should refer to it in the comment.